config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
Yes, it would also be nice if I didn't have to lock and unlock my apartment door every time I come and go. But I do it anyway, because the small inconvenience definitely outweighs all the downsides of not doing it...

Is typing `rake styleguide` such a big hassle? We can even change the default port to something that's not the rails default.